src/test/java/com/puppycrawl/tools/checkstyle/internal/XdocsJavaDocsTest.java

```diff
@@ -113,6 +113,12 @@ public class XdocsJavaDocsTest extends AbstractModuleTestSupport {
             "MultipleStringLiterals - ignoreOccurrenceContext",
         }).collect(Collectors.toSet()));
 
+    // We skip validation of examples section on modules that have this section generated
+    // until https://github.com/checkstyle/checkstyle/issues/13100
+    private static final Set<String> MODULES_EXAMPLES_TO_SKIP = Set.of(
+        "WhitespaceAfter"
```

Comment:
Think this point is getting lost.
https://github.com/checkstyle/checkstyle/pull/13357#discussion_r1271303812
> I am not seeing where it is updated, and I still think these are 2 separate activities and can be worked on separately. It should be pretty easy to gut xdocjavadoc of examples even while https://github.com/checkstyle/checkstyle/issues/13100 is in progress. https://github.com/checkstyle/checkstyle/issues/13100 is just converting examples in xdocs and has no bearing on javadoc.

Reply:
I opened https://github.com/checkstyle/checkstyle/issues/13429 for this. Changing to `until https://github.com/checkstyle/checkstyle/issues/13429`